Create rule S6414[terraform]: Excluding users or groups activities from audit logs is security-sensitive (#805)
SonarSource/rspec: rules/S6414/description.adoc

> The Google Cloud audit logs service records administrative activities and accesses to Google Cloud resources of the project. It's important to enable audit logs to maintain compliance with security standards and to assist in the event of a security incident. 

Reviewer: I thin we should not mention "security standards" unless we can reference a precise one.
```suggestion
The Google Cloud audit logs service records administrative activities and accesses to Google Cloud resources of the project. It's important to enable audit logs to be able to investigate malicious activities in the event of a security incident. 
```

> Some project members may be exempt from logging their activity in the Google Cloud audit log service, creating a blind spot and reducing the possibility of full investigations.

Reviewer: ```suggestion
Some project members may be exempted from having their activities recorded in the Google Cloud audit log service, creating a blind spot and reducing the capacity to investigate future security events.
```
